components/Footer.jsx

```diff
@@ -1,11 +1,64 @@
-import React from 'react'
+import { FOOTER_LINKS, SOCIAL_CONTACT_DETAILS } from "@/utils/constants";
+import Image from "next/image";
+import React from "react";
+import Logo from "./Logo";
 
 const Footer = () => {
   return (
-    <div>
-      This is footer section
+    <div className="w-full">
+      <div className="w-full bg-[#141311]  text-white px-10 py-16 flex md:flex-row flex-col md:gap-0 gap-5 border-b-2">
+
+      <div className="flex flex-col md:w-[30%] items-center md:items-start justify-center md:justify-start w-full gap-4 md:px-10 ">
+        <div className="flex gap-3">
+          <Image
+            src={"/logo.png"}
+            alt=""
+            width={720}
+            height={1020}
+            className="w-auto h-auto aspect-square"
+          />
+          <Logo />
+        </div>
+
+        <p className="text-lg flex justify-center items-center">
+          Fashion: Where self expression meets creativity, embracing diversity
+          and individuality.
+        </p>
+        <div className="flex flex-row items-center gap-5">
+          {SOCIAL_CONTACT_DETAILS.map((profile, index) => {
+            return (
+              <div key={index} className="">
+                {profile.icon}
+              </div>
+            );
+          })}
+        </div>
+      </div>
+
+      <div className="flex md:flex-row flex-col md:w-[70%] w-[100%]  justify-around items-center md:items-start md:gap-0 gap-10">
```

Comment:
```suggestion
      <div className="flex md:flex-row flex-col md:w-[70%] w-full  justify-around items-center md:items-start md:gap-0 gap-10">
```

Make sure this is done through out the code